awx/main/conf.py

```diff
@@ -137,34 +137,34 @@
     allow_blank=True,
     encrypted=True,
     read_only=False,
-    label=_('Red Hat customer password'),
-    help_text=_('This password is used to send data to Automation Analytics'),
+    label=_('Red Hat Client Secret for Analytics'),
+    help_text=_('This client secret is used to send data to Automation Analytics'),
     category=_('System'),
     category_slug='system',
 )
 
 register(
-    'SUBSCRIPTIONS_USERNAME',
+    'SUBSCRIPTIONS_CLIENT_ID',
```

Comment:
Let me talk this out loud - there is no way to migrate old->new via a data migration, is that correct?

So we're just going to drop the old settings.

Reply:
correct no way to migrate this -- user will need to go get their client ID and client secret from console.redhat.com and enter it into the fields